src/main/java/io/percy/appium/Environment.java

```diff
@@ -4,7 +4,7 @@
 
 public class Environment {
     private AppiumDriver driver;
-    public static final String SDK_VERSION = "2.0.0-beta.0";
+    public static final String SDK_VERSION = "2.0.2-beta.1";
```

Comment:
this would be 2.0.1-beta.1, in pom.xml we specify which the future version would be. this is current released version

Reply:
updated.